docs/source/simulation_cli.rst

```diff
@@ -21,45 +21,269 @@ simulate. The result of the simulation is stored, after a succesful
+.. option:: --n_prior_included N_PRIOR_INCLUDED
+
+    The number of prior included papers. Only used when :code:`prior_idx` is not given. Default 1.
+
+.. option:: --n_prior_excluded N_PRIOR_EXCLUDED
+
+    The number of prior excluded papers. Only used when :code:`prior_idx` is not given. Default 1.
+
+
+.. option:: --prior_idx [PRIOR_IDX [PRIOR_IDX ...]]
+
+    Prior indices by rownumber (0 is first rownumber).
+
+
+.. option:: --init_seed INIT_SEED
+
+    Seed for setting the prior indices if the prior_idx option is not used. If the option
+    prior_idx is used with one or more index, this option is ignored.
+
+
+
+Simulation Set-up
+~~~~~~~~~~~~~~~~~
+
+.. option:: --n_instances N_INSTANCES
+
+  The ``--n_instances`` argument controls the number of records that have to be
   labeled before the model is retrained, and is set at 1 by default. If
   you want to reduce the number of training iterations, for example to limit the
   size of your state file and the time to simulate, you can increase
-  ``--n_instances``.
+  ``--n_instances``. Default 1.
+
+.. option:: --stop_if STOP_IF
+
+    The number of label actions to simulate. Default, 'min' will stop
+    simulating when all relevant records are found. Use -1 to simulate all
+    labels actions.
+
+.. option:: -w WRITE_INTERVAL, --write_interval WRITE_INTERVAL
+
+    The simulation data will be written away after each set of thismany
+    labeled records. By default only writes away data at the endof the
+    simulation to make it as fast as possible.
+
+
+Save
+~~~~
+
+.. option:: --config_file CONFIG_FILE
+
+    Configuration file with model settings and parameter values.
+
+
+.. option:: --state_file STATE_FILE, -s STATE_FILE
+
+    Location to ASReview project file of simulation.
+
+
+
+
+Algorithms
+----------
+
+:program:`asreview algorithms` provides an overview of all available active
+learning model elements (classifiers, query strategies, balance
+strategies, and feature extraction algorithms) in ASReview.
+
+.. code:: bash
+
+    asreview algorithms
+
+.. note::
+
+    :program:`asreview algorithms` included models added via extensions.
+    See :ref:`develop-extensions` for more information on extending ASReview with new
+    models via extensions.
+
+
+The following components of the active learning model are available:
+
+.. note::
+
+  Some classifiers (models) and feature extraction algorithms require additional dependecies. Use :code:`pip install asreview[all]` to install all additional dependencies at once.
+
+
+.. _feature-extraction-table:
+
+Feature Extraction
+~~~~~~~~~~~~~~~~~~
+
++----------------+-----------------------------------------------------------+-----------------------------------------------------------------------------+
+| Name           | Reference                                                 | Requires                                                                    |
```

Comment:
I propose to remove all these tables. See #1204 1204

Reply:
ok!